packages/ifpack2/src/Ifpack2_LocalFilter_def.hpp

```diff
@@ -79,7 +79,7 @@ bool
 LocalFilter<MatrixType>::
 mapPairIsFitted (const map_type& map1, const map_type& map2)
 {
-  return Tpetra::Details::isLocallyFitted (map1, map2);
+  return Tpetra::isLocallyFitted (map1, map2);
```

Comment:
I'm OK with promoting this to the public interface of Tpetra, as long as you like the name.